js/setup.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var NAMES = ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var SURNAMES = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var COAT_COLORS = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var EYE_COLORS = ['black', 'red', 'blue', 'yellow', 'green'];
+var WIZZARDS_COUNT = 4;
+
+var setup = document.querySelector('.setup');
+var setupSimilar = document.querySelector('.setup-similar');
+var similarListElement = document.querySelector('.setup-similar-list');
+var similarWizardTemplate = document.querySelector('#similar-wizard-template').content.querySelector('.setup-similar-item');
+
+// Случайное число
+var getRandomInt = function (min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+// Формируем одного волшебника
+var createWizzard = function () {
+  return {
+    name: NAMES[getRandomInt(0, NAMES.length - 1)] + ' ' + SURNAMES[getRandomInt(0, SURNAMES.length - 1)],
+    coatColor: COAT_COLORS[getRandomInt(0, COAT_COLORS.length)],
+    eyesColor: EYE_COLORS[getRandomInt(0, EYE_COLORS.length)]
```

Comment:
для eyesColor и coatColor нужно скорректировать на 1, иначе цвет будет undefined, можно потестить дебаггером
```suggestion
    eyesColor: EYE_COLORS[getRandomInt(0, EYE_COLORS.length - 1)]
```

Reply:
Готово